frontend/app/src/locales/en.json

```
@@ -2244,6 +2244,22 @@
       "name": "crypto.com mobile app",
       "note": "Important notes for importing data from {0}'s CSV exports."
     },
+    "blockfi": {
+      "import": "Import {0} from BlockFi",
+      "import_trade": "trades",
+      "import_transactions": "transactions",
+      "name": "BlockFi",
+      "line_one": "[Experimental] We have developed this importer with few information. If you find an issue with your data, let us know so we can adapt the code accourding.",
+      "line_two": "BlockFi let's you export two different files. Trades will be imported from the file that only contains trades.",
```

I don't really understand this comment

```suggestion
      "line_two": "BlockFi lets you export two different files. Trades will be imported from the file that only contains trades.",
```